youtube_dl/extractor/crunchyroll.py

```diff
@@ -603,6 +606,13 @@ class CrunchyrollShowPlaylistIE(CrunchyrollBaseIE):
         'url': 'http://www.crunchyroll.com/ladies-versus-butlers?skip_wall=1',
         'only_matching': True,
     }]
+    # Is possible make test, that try URL, and return: This URL is invalid for this extractor
+    '''
+    , {
+        'url': 'http://www.crunchyroll.com/media-723735',
+        'invalid_url': True,
+    }
+    '''
```

Comment:
What you you doing?

Reply:
You don't need to. This is checked automatically once URL is included in test set of some other extractor.